CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+## v0.3.0 [2022-07-20]
+
+_What's new?_
+
+- New table added
+  - [steampipecloud_organization_workspace_member](https://hub.steampipe.io/plugins/turbot/steampipecloud/tables/steampipecloud_organization_workspace_member) ([#19](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/19))
+
+_Enhancements_
+
+- Updated: Rename column `mod_constraint` to `constraint` in `steampipecloud_organization_workspace_mod` table ([#19](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/19))
+- Updated: Rename column `workspace_state` to `state` in `steampipecloud_workspace` table ([#19](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/19))
+  - workspace_state to state
+- Added the following new columns to `steampipecloud_workspace` table: ([#19](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/19))
+  - created_by
+  - updated_by
+  - version_id
+- Added the following new columns to `steampipecloud_organization_member` table: ([#19](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/19))
+  - created_by
+  - updated_by
+  - org_handle
+- Added Get method to `steampipecloud_organization_member` table for getting a particular org member ([#19](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/19))
+- Added the following new columns to `steampipecloud_organization` table: ([#19](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/19))
+  - created_by
+  - updated_by
+- Added Get method to `steampipecloud_organization` table for getting a particular organization ([#19](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/19))
+
+_Dependencies_
+
+- Upgraded [steampipe-cloud-sdk-go](https://github.com/turbot/steampipe-cloud-sdk-go/blob/main/CHANGELOG.md#012-2022-07-19) to v0.1.2 ([#19](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/19))
```

Comment:
Can you please update this note to use the same language that we use in other `Dependencies` section, e.g., https://github.com/turbot/steampipe-plugin-oci/blob/main/CHANGELOG.md#v0160-2022-07-14?